Resources/views/Menu/sonata_menu.html.twig

```diff
@@ -33,6 +33,10 @@
             {%- endfor %}
         {%- endif %}
 
+        {%- if item.extra('route') is not empty and item.extra('route') == request.get('_route') %}
+            {%- set active = true %}
+        {%- endif %}
+
```

Comment:
> Why it should be in model?

To simply test it and simplify view logic.

Looking deeper in the code, I think the whole block should be in a MenuVoter instead.
But this should be an other PR, so never mind ;-)